lib/logdna/resources.rb

```diff
@@ -6,8 +6,9 @@ module Resources
   MAX_REQUEST_TIMEOUT = 300000
   MAX_LINE_LENGTH = 32000
   MAX_INPUT_LENGTH = 80
+  BACKOFF_PERIOD = 60
   FLUSH_INTERVAL = 0.25
-  TIMER_OUT = 15
+  TIMER_OUT = 120
```

Comment:
Why `120` instead of `30` and `TIMER_OUT` instead of `TIMEOUT` or `REQUEST_TIMEOUT`?

Reply:
> Let's try to make it consistent with [`Node.js Code Library`](https://github.com/logdna/nodejs) regarding the configurable options...

@smusali  yes I had this idea but I am not sure how safe it is to change the names of the configurable options. If we change it, it make break the users' code.

Reply:
As you can see, I did not give the name to this variable.

Reply:
`TIMER_OUT` is not such variable that customer touches while using but in general, you are right